practice/blackjack/main.py

```diff
@@ -0,0 +1,147 @@
+from hand import Hand
+from card import Card
+from deck import Deck
+from random import shuffle
+
+NAME_CARDS = ['j', 'q', 'k']
+NUMBER_CARDS = ['2', '3', '4', '5', '6', '7', '8', '9', '10']
+SUITS = ['h', 'd', 's', 'c']
+RANKS = NAME_CARDS + NUMBER_CARDS + ['a']
+
+
+def set_up_game():
+    """Creates deck, and sets up player with hand of two cards."""
+    hand = Hand([])
+    deck = create_deck()
+    game_over = False
+    hand.card_list += [draw_card_from_deck(deck)]
+    hand.card_list += [draw_card_from_deck(deck)]
+    print('Welcome to Blackjack Deathmatch. My name is Skynet, and I am your dealer. '
+          'Your hand is {}, {}'.format(hand.card_list[0], hand.card_list[1]))
+    return deck, hand, game_over
+
+
+def prompt_for_hit(hand, deck, game_over):
+    """Prompts the user if they would like to hit, then runs hit player function if yes.
+    # Split into smaller parts?
+    """
+    while game_over is False:
+        # hit_prompt = input('Would you like to hit? (y/n) ')
+        hit_prompt = 'n'
+        if hit_prompt == 'y':
+            hit_player(hand, deck)
+            score = score_hand(hand)
+            game_over = check_score(score)
+            print('Your new score is {}'.format(score))
+        elif hit_prompt == 'n':
+            score = score_hand(hand)
+            print('Alright, your final score is {}'.format(score))
+            game_over = True
+        if game_over is True:
+            break
```

Comment:
This `break` statement is a no-op. The loop condition already checks this and will break.